packages/web/app/api/webhook-healthcheck/route.ts

```diff
@@ -0,0 +1,76 @@
+import { NextRequest } from 'next/server'
+import { z } from 'zod'
+
+const AddressSchema = z.custom<`0x${string}`>(
+  val => typeof val === 'string' && /^0x[0-9a-fA-F]{40}$/.test(val),
+  'invalid evm address'
+)
+
+const KongWebhookSchema = z.object({
+  abiPath: z.string(),
+  chainId: z.number(),
+  address: AddressSchema,
+  blockNumber: z.bigint({ coerce: true }),
+  blockTime: z.bigint({ coerce: true }),
+  subscription: z.object({
+    id: z.string(),
+    url: z.string().url(),
+    abiPath: z.string(),
+    type: z.enum(['timeseries']),
+    label: z.string()
+  })
+})
+
+const OutputSchema = z.object({
```

Comment:
can we pull the main one over to `ingest`? So we have less duplication and the chances of this being stale!

Reply:
good catch! i did this on purpose actually. this way anyone can just copypaste from this route to start their own hook. the schemas could drift youre right, but these two in particular should be pretty stable